mypy/build.py

```diff
@@ -2252,6 +2259,7 @@ def find_module_and_diagnose(manager: BuildManager,
       caller_line: the line number of the import
       ancestor_for: the child module this is an ancestor of, if applicable
       root_source: whether this source was specified on the command line
+      quick: skip any error diagnosis and reporting
```

Comment:
It's kind of odd to have a flag that suppresses the `and_diagnose` part of the function's name. :-) Also `quick` reminds (me) of `--quick-and-dirty` but it has nothing to do with that. Also might want to clarify that `ModuleNotFound` is still raised. (Hm, the docstring should probably mention that that is raised in the first place?)

Reply:
OK, I will rename it to `skip_diagnose` to be more explicit.